microfit/analysis.py

```diff
@@ -679,14 +687,42 @@ def two_hypothesis_test(
         print("Generating H0 histogram")
         # generate the multiband histogram
         h0_hist = self.generate_multiband_histogram(
-            include_multisim_errors=True, use_sideband=True, scale_to_pot=scale_to_pot
+            include_multisim_errors=True, use_sideband=True, add_precomputed_detsys=add_precomputed_detsys, scale_to_pot=scale_to_pot
         )
         print("Generating H1 histogram")
         self.set_parameters(h1_params, check_matching=True)
         h1_hist = self.generate_multiband_histogram(
-            include_multisim_errors=True, use_sideband=True, scale_to_pot=scale_to_pot
+            include_multisim_errors=True, use_sideband=True, add_precomputed_detsys=add_precomputed_detsys, scale_to_pot=scale_to_pot
         )
 
+        ''' 
```

Comment:
As a general comment I'd prefer it if we did not leave commented-out code inside the files. Either we keep some code or we don't. If something is temporarily broken but to be fixed in the future, then we can also raise a NotImplementedError.